LPD-Nomelosé

victorg1991 opened the pull request.

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

sandrodw3 commented:
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/84673. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

sandrodw3 committed:
LPD-58671 Add autoTranslateURL

sandrodw3 committed:
LPD-58671 Use selectedLocaleLabel

sandrodw3 committed:
LPD-58671 Send event for auto translation

sandrodw3 committed:
LPD-58671 Make auto translation work on localized inputs

sandrodw3 committed:
LPD-58671 Adapt rich text fragment

victorg1991 committed:
LPD-62179 This has the correct type already

When enableObjectEntryVersioning is enabled in the object it creates a new version when publishing. This version is using the object entry dto as a base which already have the correct type for the Picklist and multiselectPicklist. Without this it tries to cast it to the wrong type and fail

victorg1991 committed:
LPD-62790 Include config autoTranslationEnabled

victorg1991 committed:
LPD-62793 Include data-localizable and data-field-type

victorg1991 committed:
LPD-62793 Remove unneeded quote

sandrodw3 committed:
LPD-58671 buildLang

sandrodw3 committed:
LPD-62793 We don't need to add the data-attribute anymore

sandrodw3 committed:
LPD-58671 Adapt select from list fragment

sandrodw3 committed:
LPD-58671 Adapt multiple select from list fragment

sandrodw3 committed:
LPD-58671 Sort

sandrodw3 committed:
LPD-58671 Disable options when it applies

sandrodw3 committed:
LPD-58671 Adapt existing tests

sandrodw3 committed:
LPD-58671 Add new playwright test

sandrodw3 committed:
LPD-63422 Make default fields required

sandrodw3 committed:
LPD-63422 Mark title as name field

sandrodw3 committed:
LPD-63422 Add new playwright test

sandrodw3 committed:
LPD-63422 Adapt jest tests

sandrodw3 committed:
LPD-62179 Add test

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.
